HelpMyStreetFE/HelpMyStreetFE/Helpers/ShiftRequestExtensions.cs

```diff
@@ -4,24 +4,40 @@
 using System.Linq;
 using System.Threading.Tasks;
 using HelpMyStreet.Utils.Enums;
+using HelpMyStreet.Utils.Extensions;
 using HelpMyStreet.Utils.Models;
 
 namespace HelpMyStreetFE.Helpers
 {
     public static class ShiftRequestExtensions
     {
-        public static string TimeSpan(this ShiftJob shiftJob)
+        public static string DateSpan(this ShiftJob shiftJob)
         {
             if (shiftJob.StartDate.Date.Equals(shiftJob.EndDate.Date))
             {
-                return $"{shiftJob.StartDate:dd/MM/yyyy HH:mm}-{shiftJob.EndDate:HH:mm}";
+                return $"{shiftJob.StartDate.ShiftyDate()}";
```

Comment:
This if block was to display the end date when different to the start date.  If we're no longer doing that, the if block can be removed.